package.json

```diff
@@ -100,13 +100,21 @@
       "default": true,
       "order": 7
     },
-    "useFlowParser": {
-      "title": "Use Flow Parser",
-      "description": "Use the [flow](https://github.com/facebook/flow) parser instead of [babylon](https://github.com/babel/babylon).",
+    "jsxBracketSameLine": {
+      "title": "JSX Bracket Same Line",
+      "description": "Put > on the last line in JSX",
```

Comment:
The title and the description seem to represent opposite states. Does `false` mean it puts it on the same line or the last line?